docs/proposals/deploy-csi-to-k3s.md

```diff
@@ -0,0 +1,54 @@
+# Proposal: Deploy CSI on K3s cluster 
+
+Last updated: 8.04.22
+
+
+## Abstract
+
+Deploy CSI on K3s cluster, test it work and add steps for manual patching scheduler 
```

Comment:
Purpose of this document is to specify all changes that are required to support new platform and describe installation process

Reply:
@mishoyama Do we want to describe both automated installation and manual patching in this proposal?